apps/ledger-live-mobile/src/families/elrond/components/Details/Details.tsx

```diff
@@ -69,8 +70,6 @@ const Details = (props: DetailsPropsType) => {
           {Boolean(amount) && (
             <Section title={t("operationDetails.extra.delegatedAmount")} value={amount} />
           )}
-
-          {extra.memo && <Section title={t("operationDetails.extra.memo")} value={extra.memo} />}
```

Comment:
We don't display the `memo` Section anymore?

Reply:
Same as for Celo...